documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -95,7 +95,13 @@ Capacity limits can be set for the following broker resources:
 For disk storage, use a number value with either standard Kubernetes byte units (K, M, G, or T), their bibyte (power of two) equivalents (Ki, Mi, Gi, or Ti), or a byte value with or without E notation.
 For network throughput, use an integer value with standard Kubernetes byte units (K, M, G) or their bibyte (power of two) equivalents (Ki, Mi, Gi) per second.
 
-Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
+For Kafka clusters that do not use JBOD storage, Strimzi generates a generic Cruise Control capacity configuration entry that applies to every broker.
+
+For Kafka clusters that use JBOD storage, do not set a disk capacity limit.
+Strimzi generates a specific disk capacity configuration for this type of cluster. 
+This is required because the path names of the Kafka JBOD volumes include a broker ID. 
+The configuration allows for intra-broker disk balancing.
+If you set a disk capacity limit, it will override the capacity configuration for the cluster with a standard configuration.
```

Comment:
Why does it sound as with with one disk, user has to configure something manually while with multiple disks we configure it? Seems strange and user-unfriendly.

Reply:
Strimzi will auto configure the storage capacity config in all case whether it running a Kafka cluster deployment using non-JBOD, JBOD with multiple disks, JBOD with single disk. What I am trying to articulate is that we don't want users manually setting the disk capacity when using JBOD storage since it will override the special JBOD config generated by Strimzi with something like this:
```
{
  "brokerCapacities":[
    {
      "brokerId": "-1",
      "capacity": {
        "DISK": <Manually set disk capacity value>,
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "This is the default capacity. Capacity unit used for disk is in MB, cpu is 
              in percentage, network throughput is in KB."
    }
  ]
}
```
which will prevent intra-broker balancing

Reply:
Makes me think now that maybe we should ditch this explanation and just prevent users from setting the disk capacity when using JBOD storage. It doesn't make sense to allow them to override it in the first place as it will only limit the features they can use

Reply:
It will be easier for me to fix the code to prevent this then explaining in the docs for users to avoid it :D

Reply:
Why would people configure the disk in the first place when we generate the configuration automatically? Why would I care when you use the _generic_ configuration and when you use the per-broker configuration? I mean, this is a part of Strimzi I do not understand much. And I do not think this really helps me to understand the difference and what it means for me.

Reply:
> Makes me think now that maybe we should ditch this explanation and just prevent users from setting the disk capacity when using JBOD storage. It doesn't make sense to allow them to override it in the first place as it will only limit the features they can use

I'm not sure. Why would the user want to configure the disk capacity with a single disk? And if they want to configure it for a single disk, why would they not want that for multiple disks?

Reply:
I mean, the text right now sounds to me like user of single disk has to configure more manually than a user with multiple disks. But you now make it sounds like user with a single disk has actually more options. But it is not clear why these options matter.

Reply:
> Why would people configure the disk in the first place when we generate the configuration automatically?

You are right, I confused myself on this. Users shouldn't configure the disk because Strimzi automatically generates it. And even if they set the disk capacity the generated JBOD configuration will be unaffected/safe so all we can remove this whole paragraph!